mne/channels/montage.py

```diff
@@ -668,19 +668,11 @@ def transform_to_head(montage):
     # Get fiducial points and their coord_frame
     native_head_t = compute_native_head_t(montage)
     montage = montage.copy()  # to avoid inplace modification
-
     if native_head_t['from'] != FIFF.FIFFV_COORD_HEAD:
         for d in montage.dig:
             if d['coord_frame'] == native_head_t['from']:
                 d['r'] = apply_trans(native_head_t, d['r'])
                 d['coord_frame'] = FIFF.FIFFV_COORD_HEAD
-            elif d['kind'] == FIFF.FIFFV_POINT_EEG:
-                raise RuntimeError(
```

Comment:
Instead of reverting, can/should we now consider a `_on_missing` style behavior, perhaps with a default of `on_frame_mismatch='warn'` (and what you had here would have been equivalent to `on_mismatch='raise'`)? MNE-BIDS could inspect the `_get_args` and see if `'on_frame_mismatch'` is in there, and add `on_frame_mismatch='ignore'`

Reply:
I'd rather fix MNE-BIDS and then bring back the error ;)